x/bank/README.md

```diff
@@ -595,6 +592,11 @@ The bank module contains the following parameters
 The SendEnabled parameter is now deprecated and not to be use. It is replaced
 with state store records.
 
+:::warning
+Note, SendEnabled is depreacted and only kept for backward compatibility.
```

Comment:
```suggestion
Note, SendEnabled is deprecated and only kept for backward compatibility.
```

And I see above line statement resembles the same. Try to merge both.

Reply:
oh shoot forgot to remove that, i meant to make this popout more with the warning